pkgs/applications/editors/poke/default.nix

```
@@ -97,18 +75,17 @@ in stdenv.mkDerivation rec {
       # Expect the text in format of '<a href="...">poke 2.0</a>'
       new_version="$(curl -s https://www.jemarch.net/poke |
           pcregrep -o1 '>poke ([0-9.]+)</a>')"
-      update-source-version ${pname} "$new_version"
+      update-source-version poke "$new_version"
     '';
   };
 
   meta = with lib; {
```

It deserves a removal (https://github.com/NixOS/nixpkgs/issues/208242)

```suggestion
  meta = {
```